refs/biblio.json

```diff
@@ -1347,6 +1347,12 @@
         "status": "W3C-Internal Document",
         "publisher": "W3C"
     },
+    "LIGHTNING-VOCABULARY": {
```

Comment:
This is the International _*Lighting*_ Voc, not Lightning. The key above should be changed, otherwise this will be confusing!